### PR TITLE
Slider Component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my slider component

this page contains the slider component

live link: https://css-companion.netlify.app/components/slider.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of slider components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media

image

### CODE REVIEW

File: component/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+
+main{
+    
+    /* grid-template-areas: 200px 1fr; */
+    /* align-items: center; 
+    padding: 3.5rem;
+    text-align: center;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin: auto; */
```
Comment (sahiba0915) [inline component/slider/slider.css]: why is this so? if you did'nt want this you can remove

File: component/slider/slider.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./slider.css">
+
+    <script src="https://kit.fontawesome.com/e45a62e93a.js" crossorigin="anonymous"></script>
+</head>
+    <title>Document</title>
```
Comment (sahiba0915) [inline component/slider/slider.html:11]: give a proper title

**Review by sahiba0915 — COMMENTED** [review]
looks good to go

File: component/slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
```
Comment (pnchinmay) [inline component/slider/slider.css]: color variables are preferred

File: component/slider/slider.css [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+aside {
+
+    background: #111827;
```
Comment (pnchinmay) [inline component/slider/slider.css:113]: same here

Comment (pnchinmay) [reply component/slider/slider.css:113]: Also, remove the empty lines within the code blocks

**Review by pnchinmay — APPROVED** [review]
Left some suggestions
Everything else looks good :100:

File: component/slider/slider.css (hunk 1 above)
Comment (i-am-a-codegirl) [reply component/slider/slider.css]: yes

**Review by sahiba0915 — APPROVED** [review]

**Pull request merged**